packages/server/src/backup/backup.service.ts

```diff
@@ -101,6 +105,41 @@ export class BackupService {
     }
   }
 
+  // Daily Uploads Backup Task - Keeps rolling backups for 5 days
+  @Cron(CronExpression.EVERY_DAY_AT_MIDNIGHT) //tentative
+  async handleDailyUploadsBackup() {
+    try {
+      const date = new Date().toISOString().split('T')[0];
+      const uploadsDir = './uploads';
+      const backupFile = `uploads_backup-${date}.tar.gz`;
+      const backupDir = '../../backups/uploads-daily';
+
+      const hasSpace = await this.checkDiskSpace(backupDir);
+      if (!hasSpace) {
+        console.error('Insufficient disk space for uploads backup.');
+        Sentry.captureMessage('Insufficient disk space for uploads backup.');
+        return;
+      }
+
+      // Use `tar` to compress the uploads directory
+      const compressCommand = `tar -czf ${backupDir}/${backupFile} -C ${uploadsDir} .`;
+
+      exec(compressCommand, (error, stdout, stderr) => {
+        if (error) {
+          console.error(`Uploads backup failed: ${stderr}`);
+          Sentry.captureMessage(`Uploads backup failed: ${stderr}`);
+        } else {
+          console.log(`Uploads backup saved: ${backupFile}`);
+          // Retain only the last 5 backups
+          this.deleteOldBackups(backupDir, 5); //tentative
```

Comment:
and then keep 14 days of backups

Reply:
Neither 3, 4, or 5 day gaps between backups would be divisible if I kept 14 days of backups. I'll opt for every 4 days, and I'll keep the last 3 backups (12 days). Let me know if this sounds good to you.

Reply:
sure yeah 12 days should be fine